src/fides/api/common_exceptions.py

```diff
@@ -25,6 +25,14 @@ class TraversalError(FidesopsException):
     """Fidesops error with the names of all nodes that could not be reached."""
 
 
+class UnreachableNodesError(TraversalError):
+    """Fidesops error with the names of all nodes that could not be reached."""
+
+
+class UnreachableEdgesError(TraversalError):
+    """Fidesops error with the names of all edges that could not be reached."""
+
```

Comment:
These both inherit from `TraversalError` so any code that's currently catching the `TraversalError` exception should continue to work as before. I added these more specific ones just to be able to know the specific reason for the traversal error, in order to show a more accurate error message in the `ExecutionLog`